citrination_client/data/dataset_file.py

```diff
@@ -0,0 +1,40 @@
+class DatasetFile(object):
+    """
+    Class representation of a file in a dataset on Citrination.
+    """
+
+    def __init__(self, path, url=None):
```

Comment:
Where did you land on this type of documentation and how it renders in sphinx? Do the parameters have their documentation generated in a sensible way?

Reply:
They do, I told it to not skip `__init__` methods

Reply:
![image](https://user-images.githubusercontent.com/15318848/37982937-c7ec5e62-31a6-11e8-996a-c04be0fd57b6.png)